python/src/aiconfig/default_parsers/parameterized_model_parser.py

```diff
@@ -45,7 +45,7 @@ async def run(
         aiconfig: AIConfig,
         options: Optional[InferenceOptions] = None,
         parameters: Dict = {},
-        **kwargs,
+        **kwargs, #TODO: We should remove and make arguments explicit
```

Comment:
We need to add `run_with_dependencies` arg here otherwise it will fail. Ex: https://github.com/lastmile-ai/aiconfig/blob/b7b9910cc7abbd9b08dbdfe634e25f98396afcef/python/src/aiconfig/editor/server/server.py#L381

It's weird that your test plan showed this working, this should've failed. Can you do `pip list | grep aiconfig` and make sure that your aiconfig is linked to local and not the production package? If it's not seeing `python-aiconfig` to a local path for you, you need to go to `aiconfig/python` and run `pip install -e .`